web-app/js/portal/ui/MapPanel.js

```diff
@@ -109,26 +107,6 @@ Portal.ui.MapPanel = Ext.extend(Portal.common.MapPanel, {
         this.map.setDefaultSpatialConstraintType(this.defaultSpatialConstraintType);
     },
 
-    zoomToLayer: function(openLayer) {
-        if (openLayer && openLayer.hasBoundingBox()) {
-            // build openlayer bounding box
-            var bounds = null;
-            if (openLayer.bboxMinY == -180 && openLayer.bboxMaxY == 180 && this.enableDefaultDatelineZoom) {
```

Comment:
This code was introduced in eef0d5b79f25.  It appears as though `enableDefaultDatelineZoom` is never set to `true` anymore, hence I don't *think* we need this logic.